manage_arkime/core/capacity_planning.py

```diff
@@ -0,0 +1,93 @@
+from dataclasses import dataclass
+import math
+import logging
+
+logger = logging.getLogger(__name__)
+
+INSTANCE_TYPE_CAPTURE_NODE = "m5.xlarge" # Arbitrarily chosen
+TRAFFIC_PER_M5_XL = 2 # in Gbps, guestimate, should be updated with experimental data
+MAX_TRAFFIC_PER_CLUSTER = 100 # Gbps, scaling limit of a single User Subnet VPC Endpoint
+MINIMUM_NODES = 1 # We'll always have at least one capture node
+MINIMUM_TRAFFIC = MINIMUM_NODES * TRAFFIC_PER_M5_XL
```

Comment:
why don't the min and max names match? Seems like they should either be MIN_TRAFFIC_PER_CLUSTER or MAXIMUM_TRAFFIC or am I missing one has units and the other doesn't?

Reply:
Hmm, I vaguely remember there being a reason at some point last week, but I don't know what it is now.  I'll update to `MAXIMUM_TRAFFIC`.